src/settings.cmake

```diff
@@ -247,3 +251,33 @@ if (WIN32 AND (CLI_CMAKE_PLATFORM_ARCH_ARM OR CLI_CMAKE_PLATFORM_ARCH_ARM64))
 	      message("Using Windows SDK version ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}")
       endif()
 endif ()
+
+if (WIN32)
+    if(CLI_CMAKE_PLATFORM_ARCH_ARM)
+      # Explicitly specify the assembler to be used for Arm32 compile
+      if($ENV{__VSVersion} STREQUAL "14 2015")
```

Comment:
Does "14 2015" need to be updated?

Reply:
We now officially support only VS 2017 and VS 2019.